cmd/limactl/list.go

```diff
@@ -94,6 +100,9 @@ func listAction(cmd *cobra.Command, args []string) error {
 	if quiet && jsonFormat {
 		return errors.New("option --quiet conflicts with --json")
 	}
+	if message && jsonFormat {
+		return errors.New("option --message conflicts with --json")
+	}
```

Comment:
`--message` also does conflict with `--quiet` (as does `--format`, so we missed that as well before).

Not sure if it is worth checking for all those combination though; we could just say that `--quiet` takes precedence over the others... 😄

Reply:
And --quiet is somewhat weird anyway, it is usually just the opposite of --verbose. Here it is more like --name-only

Reply:
Yes, `--format`, `--json`, `--quiet`, and `--message` are all alternative output formats, and you obviously can only choose one of them. So `--quiet` is indeed a weird name, and maybe we should introduce an alias (`--name`?), and mark `--quiet` as "hidden"?